pkg/redo/reader/reader.go

```diff
@@ -207,11 +208,13 @@ func (l *LogReader) runReader(egCtx context.Context, cfg *readerConfig) error {
 			}
 		case redo.RedoDDLLogFileType:
 			ddl := item.data.RedoDDL
-			if ddl.DDL.CommitTs > cfg.startTs && ddl.DDL.CommitTs <= cfg.endTs {
+			// There may exist dupilicate ddls
```

Comment:
do we still may meet duplicate ddls?

Reply:
The redo reader is not modified in this pr, I will rewrite it in the future.